### DISCUSSION TITLE
A casual text game in Julia

### DISCUSSION THREAD

**Discussion by Azzaare** [Humans-of-Julia/HoJBot.jl#24] [Ideas]
Hi there!

I'm just starting a branch `gamemaster` that will rebase regularly on the main branch.

It will receive features updates from branches starting by `gm/some_feature`

The test bot is also hosted on the VPS, but on a separate account and tested on a separate discord server!

**Comment by tk3369** [comment]
What kind of game are you thinking about? I was researching on Zork (and z-machine) the other day and it's quite interesting 🧐

**Reply by Azzaare** [reply]
I don't have a specific game in mind, as I am trying to let other members influence the game content as mush as possible.

However, for traffic and visualization reasons, I think a management game (exploring, searching techs, resources managements) with some similarities with OGame is interesting.

Based on how the discussion is going, it seems that julia coding challenges might be a key feature to get resources ... ^^
So, we will probably work first on such challenges and see how it goes
